nixos/modules/virtualisation/docker-containers.nix

```diff
@@ -161,11 +161,20 @@ let
             ["--network=host"]
           '';
         };
+
+        enableOnBoot = mkOption {
```

Comment:
maybe `autoStart` (similar to https://nixos.org/nixos/options.html#containers.%3Cname%3E.autostart)

Reply:
I struggled with finding a good name for it (a programmers greatest enemy) and decided to follow https://nixos.org/nixos/options.html#virtualisation.docker.enableonboot in the end.  
Thanks for pointing it out, I forgot to ask about the name in the PR.